Program.cs

```diff
@@ -1,4 +1,16 @@
 ﻿using System;
-Console.WriteLine ("Hello, World!");
+Console.Write ("Enter a word or phrase:");
+string text = Console.ReadLine ();
+text = text.Replace (" ", "").ToLower ();
+char[] ch = text.ToCharArray ();

```

Comment:
`char[] ch = text.ToCharArray ();`
`Array.Reverse (ch);`
`string rev = new string (ch);`
This is correct. Can you write the method for ReverseString, Like below
`string ReverseString (string text)`